packages/ui-causal-graph-editor/src/shared/editor-overlay/legend/legend-data.tsx

```diff
@@ -14,58 +14,98 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { DefaultTheme } from '@darajs/styled-components';
-
 import { EditorMode } from '@types';
 
-export interface LegendLineDefinition {
+export interface LegendNodeDefinition {
+    /** defines the filled color of the node symbol */
+    color?: string;
+    /** defines the border color of the node symbol */
+    highlight_color?: string;
+}
+
+export interface LegendArrowDefinition {
```

Comment:
What's the reason to rename `Line` to `Arrow`? It's not always an arrow technically, hence the previous 'line' name - sometimes it represents just a straight line. We can just use `Edge` I guess to be consistent with the standard Node/Edge naming